hackvote/test/models/project_test.rb

```diff
@@ -0,0 +1,39 @@
+require 'test_helper'
+
+class ProjectTest < ActiveSupport::TestCase
+  
+  def setup
+  	@event = events(:example)
+  	@project = @event.projects.build(name: "foobar", creators: "me", event_id: @event.id)
```

Comment:
I don't think you need the `event_id` part here; that should be handled by `@event.projects.build`